core/src/main/java/com/graphhopper/util/PathMerger.java

```diff
@@ -86,8 +95,8 @@ public void doWork(PathWrapper altRsp, List<Path> paths, Translation tr) {
                     }
 
                     for (Instruction i : il) {
+                        origPoints += i.getPoints().size();
```

Comment:
Yes. Well it was only used for logging, so it was not really bad, also it only appeared when having multiple Paths. I thought about creating an issue for it but then I thought it's not worth to create an issue for this.